actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -617,7 +634,7 @@ public void setDeactivationTimeout(long deactivationTimeoutMs)
 
         if (hosting == null)
         {
-            hosting =  new Hosting();
+            hosting = new Hosting(localAddressCacheMaximumSize, defaultActorTTL);
```

Comment:
Based on the other comment thread. This TTL should be its own configuration to allow for tuning of the value that works best for the use case of the application that is using Orbit.